### PR TITLE
Add reloc name, wip to fix dupped relocs

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -1492,7 +1492,9 @@ static int bin_relocs(RCore *r, int mode, int va) {
 			}
 			r_cons_printf ("vaddr=0x%08"PFMT64x" paddr=0x%08"PFMT64x" type=%s",
 				addr, reloc->paddr, bin_reloc_type_name (reloc));
-			if (reloc->import && reloc->import->name[0]) {
+			if (reloc->name) {
```
Comment (ret2libc) [inline libr/core/cbin.c]: remove the whole `if` and just do a single `r_cons_printf (" %s", name);`

**Comment by radare** [comment]
yeah, its wip, and iknow its gonna be red, but its not even solving the issue of the libhellojni bin, but didnt wanted to trash  or  lose my changes

**Comment by radare** [comment]
I moved this to 3.2 because it requires some discussions and more  changes in rbin, so lets just focus on stability for the release

**Comment by radare** [comment]
@ret2libc as long as you are working on elf and bin, wanna inherit this pr?

**Comment by ret2libc** [comment]
Sure, but what's the problem exactly? Do you have an Issue, a test or a binary to reproduce the problem?

**Comment by radare** [comment]
i dont have any bin now. but can be probably easy to craft one

**Comment by ret2libc** [comment]
well this seems fixed in one of the recent PRs about relocs on ELF. Can we close?

**Comment by radare** [comment]
moving forward, dont expect/want to get changes in here a week before the rls

**Comment by radare** [comment]
did you tried setting two relocs in the same address? do we have a test related to this pr

**Comment by radare** [comment]
this will increase memory usage a lot, and this pr have leaks

**Comment by radare** [comment]
Closing, i guess this was not even a complete/proper fix, and now have conflicts and nobody seems to complain about tricked reloc hacks in here, we may handle this when we get a sample

**Pull request closed without merging**